inductiva/fluids/wind_tunnel/README.md

```diff
@@ -22,22 +22,29 @@ Launching a simulation returns a task object, which can be used to verify the st
 
 ### Example:
 
-To run this example download a vehicle geometry in OBJ format from [here](/resources/geometry/test_vehicle.obj) and
-do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
+To run this example you need an object. You can fetch any object you like directly with Inductiva package. On this example, we download a test object saved in our Github repository.
```

Comment:
Don't make the statement so generic (the user can't use "any object") we are talking about a specific file format (.obj) only -- and perhaps we should link to a reference source about this file format. In addition to that are there some constraints on the granularity of meshes, or the fact that perhaps obj files can have extra info that we won't be able to parse?

Reply:
I mentioned that above - but will clarify it here as well. Thanks! There are no constraints on the granularity... In principle, we can parse all for the simulation, but the visuals may not work.